app/components/Swapbox.tsx

```diff
@@ -36,10 +36,30 @@ export const Swapbox = () => {
       <SwapInput
         title="To Receive"
         value={tokenOutAmount}
-        onChange={(event) => {
+        onChange={event => {
           setTokenOutAmount(event.target.value);
         }}
       />
+      <section>
```

Comment:
I actually like to use `section` to divide things that are their own section. 
Because I see it as a `div`, with the same difference. Because it's handy to have that visual distinction.
But yeah, reading the docs, it should have a heading title to be a section.

Do you guys know an element that could do this?
 
**Section docs**
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Reply:
I think that's too much. I don't think I want more than `html` at this point. Not a big deal, it's just nice to have that visual division.